ProcessLib/Output.cpp

```diff
@@ -262,16 +270,12 @@ void Output::doOutputNonlinearIteration(Process const& process,
             + ".vtu";
     std::string const output_file_path = BaseLib::joinPaths(_output_directory, output_file_name);
     DBUG("output iteration results to %s", output_file_path.c_str());
-    const bool make_out =
-        !(process_id < static_cast<int>(_single_process_data.size()) - 1 &&
-          !(process.isMonolithicSchemeUsed()));
-    doProcessOutput(output_file_path, make_out, _output_file_compression,
+    doProcessOutput(output_file_path, make_output, _output_file_compression,
```

Comment:
Is it really/still necessary that `doProcessOutput()` is passed the argument `make_output`?

Reply:
Removed.

Reply:
Added it again.  Because doProcessOutput adds variable to vtu although the output can be skipped.

Reply:
I see: Writing the output file is skipped if it's not the last process.
Anyway, thanks for your effort.